target/linux/ath79/dts/qca9558_zyxel_nbg6716.dts

```diff
@@ -147,12 +147,12 @@
 
 		partition@500000 {
 			label = "kernel";
-			reg = <0x500000 0x400000>;
+			reg = <0x500000 0x600000>;
 		};
 
 		partition@900000 {
```

Comment:
@b00000

Reply:
How could this go through ...

Reply:
I've fixed this and updates the pull request.